eth2util/keymanager/keymanager.go

```diff
@@ -83,8 +86,35 @@ type keymanagerReq struct {
 	Passwords []string            `json:"passwords"`
 }
 
+// keymanagerJSON is the json formatter for keymanagerReq.
+type keymanagerReqJSON struct {
```

Comment:
This `keymanagerReqJSON` and `keymanagerReq` are too similar. Rather remove `keymanagerReq` type